<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="J">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="QN1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="QN2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="n_sweeps">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>19</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1000</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="filename">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="go">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>320</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>360</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="lattice">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lattice</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="orderparam">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Order Params</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>pdel:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>T:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>J:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Q1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Q2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>???</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="dialogueArea">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>159</width>
       <height>29</height>
      </rect>
     </property>
     <widget class="QTextBrowser" name="dialogueText">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>256</width>
        <height>192</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="labelJ">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>58</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelQN1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>58</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelQN2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>58</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelQuestion">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>58</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="N1_spin_box">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>52</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>16</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="N2_spin_box">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>52</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>4</number>
    </property>
    <property name="maximum">
     <number>32</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="R_spin_box">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>52</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>4</number>
    </property>
    <property name="maximum">
     <number>64</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial State:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="pdel_spin_box">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="T_spin_box">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <double>5.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="MainDisplay">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>241</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>482</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuThe_Model_with_no_name">
    <property name="title">
     <string>The Model with no name</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuJhaberstroh_lbl_gov_whitelam_lbl_gov">
    <property name="title">
     <string>Molecular Foundry, LBNL</string>
    </property>
   </widget>
   <addaction name="menuThe_Model_with_no_name"/>
   <addaction name="menuJhaberstroh_lbl_gov_whitelam_lbl_gov"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
